CONTRIBUTING.md

```diff
@@ -0,0 +1,62 @@
+### Contributing and Feedback:
+We welcome any input, feedback, bug reports, and contributions via the DSCI_522_Crime_Prediction_Vancouver Repository
+This outlines how to propose a change to DSCI_522_Crime_Prediction_Vancouver
+
+### How To Contribute Code to DSCI_522_Crime_Prediction_Vancouver
+Kaggle requires authentications before downloading the file. Refer to the README.md to download data via Kaggle API. 
+
+### Setting Up Your Environment
+
+Install the latest version of DSCI_522_Crime_Prediction_Vancouver locally using 
```

Comment:
We can remove the installation steps here. Contributors can follow the fork process below..

Reply:
Got it.  I updated and pushed to my forked repo. Can you confirm you can see the new changes below? (Still new to GitHub, apologies! Just double checking)